TodoParser.py

```diff
@@ -460,25 +530,18 @@ def _get_title(self, comment):
         """Check the passed comment for a new issue title (and reference, if specified)."""
         title = None
         ref = None
+        title_identifier_actual = None
         title_identifier = None
         for identifier in self.identifiers:
-            title_identifier = identifier
-            title_pattern = re.compile(fr'(?<={identifier}[\s:]).+', re.IGNORECASE)
-            title_search = title_pattern.search(comment, re.IGNORECASE)
+            title_pattern = re.compile(fr'(\b{re.escape(identifier)}\b)(\(([^)]+)\))?\s*:?\s*(.+)', re.IGNORECASE)
+            title_search = title_pattern.search(comment)
```

Comment:
This is just a better regex pattern that allows us to access the different fields by group number rather than a series of different patterns/splits/etc.